dist/models/calendar.d.ts

```diff
@@ -65,7 +70,7 @@ declare namespace Calendar {
         publicBookings: boolean;
         joinableBookings: boolean;
         hours: HoursOfOperation[];
-        privileges: Privilege[];
+        privileges?: Privilege[];
```

Comment:
Do we want to do this? If it's a collection we have to check A) existence then B) length as opposed to just checking length

Reply:
fixed